src/app/container/launch/launch.component.html

```diff
@@ -38,10 +38,15 @@
       <pre>{{ cli }}</pre>
     </div>
     <div *ngIf="(currentDescriptor | lowercase) === 'cwl'">
-      <div class="well well-sm" tooltip="Commands for launching tools through CWLtool: the CWL reference implementation.
-      cwltool can run a tool directly when all inputs and outputs are available on the local filesystem.">
-        <a href="https://github.com/common-workflow-language/cwltool">Run locally with cwltool</a>
+      <div class="well well-sm" tooltip="Commands for launching tools through cwl-runner when all inputs and outputs are available on the local filesystem.">
+        <a href="https://github.com/common-workflow-language/cwltool">Run locally with cwl-runner (you can use '--non-strict' when the executor is cwltool)</a>
         <pre>{{ cwl }}</pre>
+        </div>
+        <div class="well well-sm" tooltip="Commands for launching tools through CWLtool: the CWL reference implementation. '--non-strict' can be used.">
+        Alternatively, cwltool can run a tool off of dockstore.org when all inputs and outputs are available on the local filesystem or via HTTP(S):
```

Comment:
Same here, why would someone want to use non-strict. 
Also, what distinguishes this from the above option?

Reply:
Still working on this...